py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -152,13 +152,30 @@ def compute_expected_flux(self, forests):
         -----
         ExpectedFluxError if Forest.wave_solution is not 'lin' or 'log'
         """
-        context = multiprocessing.get_context('fork')
-        pool = context.Pool(processes=self.num_processors)
+        in_nside = 16
```

Comment:
I'm thinking we should hard-code `in_nside` at the beginning of the module so that it is made more clear. I would add
`IN_NSIDE = 16` at line 27 and then here do `in_nside = IN_NSIDE` (or use `IN_NSIDE` directly)

Reply:
Done